packages/react-liveness/src/components/FaceLivenessDetector/displayText.ts

```diff
@@ -102,6 +96,25 @@ export const defaultLivenessDisplayText: Required<LivenessDisplayText> = {
   hintIlluminationTooDarkText: 'Move to brighter area',
   hintIlluminationNormalText: 'Lighting conditions normal',
   hintHoldFaceForFreshnessText: 'Hold still',
+  photosensitivityWarningBodyText:
+    'This check flashes different colors. Use caution if you are photosensitive.',
+  photosensitivityWarningHeadingText: 'Photosensitivity warning',
+  photosensitivityWarningInfoText:
+    'Some people may experience epileptic seizures when exposed to colored lights. Use caution if you, or anyone in your family, have an epileptic condition.',
+  photosensitivityWarningLabelText: 'More information about photosensitivity',
+  photosensitivyWarningBodyText:
```

Comment:
Let's use a constant for the value here and use it for both this key and for `photosensitivityWarningBodyText` (same with the other keys prefixed with `photosensitivy`). Can be done in the same follow up as the deprecation warnings